layouts/canvas.tsx

```diff
@@ -227,9 +224,15 @@ export const CanvasLayout = ({ children }): JSX.Element => {
         <div className={styles.center}>
           <div style={{ gridAutoFlow: "row" }} className={styles.centerButton}>
             <div className={styles.centerButton}>
-              <Typography variant={"h4"} className={styles.projectName}>
-                {projectTitle || "Untitled process"}
-              </Typography>
+              {project ? (
+                <EditableTitle
+                  defaultText={project.name || "Untitled process"}
```

Comment:
Please add the correct type to the type of `Project.name`. Based on the current type `name` is a `string` meaning there's no point in having the `OR` operand.